Source/Core/DolphinLib.vcxproj

```diff
@@ -28,6 +28,10 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
+    <ClCompile Include="Core\HW\EXI\BBA\BuiltIn.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="Core\HW\EXI\BBA\BuiltIn.h" />
     <ClCompile Include="InputCommon\ControllerInterface\WGInput\WGInput.cpp" />
   </ItemGroup>
```

Comment:
Generally we put these in `DolphinLib.props` instead of `DolphinLib.vcxproj`.  WGInput should also be there; I've submitted PR #10568 to move it.

Reply:
@Pokechu22 I moved file to the .props but it's not getting compiled anymore.... im not sure what else need to be done.

Reply:
You might need to close and reopen visual studio after editing the file; it doesn't always notice changes to it.

Reply:
yes had to flush the .vs folder and restart VS